Samples/WidgetAdvSample/WidgetAdvSample.vcxproj

```diff
@@ -45,7 +45,7 @@
   </ItemGroup>
   <PropertyGroup Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
-    <PlatformToolset>v140</PlatformToolset>
+    <PlatformToolset>v143</PlatformToolset>

```

Comment:
The SDK today supports VS2017+, we'll likely want to at least support 2019 still, which is v142. We can have the project files conditionally load v142 or v143.